wordpress-mobile/MediaPicker-iOS
# Add Carousel controller delegate to present custom controllers

This PR adds a new `WPCarouselAssetsViewControllerDelegate`, where the `WPCarouselAssetsViewController` ask this delegate for a custom view controller to show assets.

This is implemented in the main app to show animated GIFs in the carousel preview. ([PR Here](https://github.com/wordpress-mobile/WordPress-iOS/pull/9621))

To test:
The easiest way to test is to check out the [PR in the main app](https://github.com/wordpress-mobile/WordPress-iOS/pull/9621) and follow the testing steps.

@SergioEstevao Would you mind to take a look at this please?
Thank you!
